CHANGELOG.md

```diff
@@ -48,6 +49,9 @@
 - Remove default features. You need to enable required features explicitly now. As a quick workaround, you may want to use the
   new `full` feature which activates all features. See [PR 2918].
 
+- New `libp2p-onion` crate
+  ([PR 2899](https://github.com/libp2p/rust-libp2p/pull/2899)).
+
```

Comment:
I added a comment to the changelog for 0.49.0. Although I'm not actually sure this pull request will be merged in time and it's not mentioned in #2931